external/runtime/runtime.depproj

```diff
@@ -48,22 +48,6 @@
     </PackageReference>
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="@(RefAndRuntimePackageReference)" />
```

Comment:
I believe there is no way to do it with BinPlaceConfigurations, but I have done this in the past by adding a target to the depproj that runs after RestorePackages that would copy the one that is not copied automatically. After RestorePackages, `@(Reference)` will contain the ref assemblies and `@(ReferenceCopyLocal)` will contain the runtime assemblies, so after just running one restore you could theoretically copy both to the right folder.